components/script/dom/mouseevent.rs

```diff
@@ -163,6 +171,16 @@ impl MouseEventMethods for MouseEvent {
         self.client_y.get()
     }
 
+    // https://drafts.csswg.org/cssom-view/#dom-mouseevent-pagex
+    fn PageX(&self) -> i32 {
+        self.page_x.get()
```

Comment:
According to [step 1](https://drafts.csswg.org/cssom-view/#dom-mouseevent-pagex), does this mean `pageX` and `pageY` should be initialized using what values are calculated for `clientX` and `clientY` since those values are relative to the origin of the viewport? Reading the definition of [initial containing block](https://drafts.csswg.org/css-display-3/#initial-containing-block) leads me to believe the the viewport and initial containing block are the same things in this case?

Reply:
Yes, we can rely on the clientX and clientY values in the short term and hopefully #24415 will make the correct values more clear.